tests/AstRunner/AstParser/NikicPhpParser/NikicPhpParserTest.php

```diff
@@ -28,12 +24,6 @@ protected function setUp(): void
         );
     }
 
-    public function testParseWithInvalidData(): void
```

Comment:
This test was removed, because it just tests for the type declaration which we can safely assume works as it is a language construct.